### PR TITLE
Add io.github.tfuxu.Halftone

### PR DESCRIPTION
**Pull request by tfuxu**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. ~~If not, I contacted upstream developers about submitting their software to Flathub. **Link:**~~
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by tfuxu** [comment]
bot, build io.github.tfuxu.Halftone

**Comment by tfuxu** [comment]
bot, build io.github.tfuxu.Halftone

File: io.github.tfuxu.Halftone.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      "/share/pkgconfig",
+      "*.la",
+      "*.a"
+  ],
+  "modules" : [
+      "pypi-dependencies.json",
+      {
+          "name" : "blueprint-compiler",
+          "buildsystem" : "meson",
+          "sources" : [
```
Comment (hfiguiere) [inline io.github.tfuxu.Halftone.json:29]:
```suggestion
          "cleanup": [
              "*"
          ],
          "sources" : [
```

File: io.github.tfuxu.Halftone.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+              {
+                  "type" : "git",
+                  "url" : "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                  "tag" : "v0.6.0"
+              }
+          ]
+      },
+      {
+        "name": "magick",
+        "buildsystem": "simple",
```
Comment (hfiguiere) [inline io.github.tfuxu.Halftone.json:39]: this should be buildsystem autotools instead.

**Comment by tfuxu** [comment]
bot, build io.github.tfuxu.Halftone

File: io.github.tfuxu.Halftone.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+              {
+                  "type" : "git",
+                  "url" : "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                  "tag" : "v0.6.0"
+              }
+          ]
+      },
+      {
+        "name": "magick",
+        "buildsystem": "autotools",
```
Comment (hfiguiere) [inline io.github.tfuxu.Halftone.json:42]: that mean not having build commands.

File: io.github.tfuxu.Halftone.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        "build-commands": [
+          "./configure --prefix=/app --with-heic=yes --with-rsvg=yes",
+          "make",
+          "make install"
+        ],
```
Comment (hfiguiere) [inline io.github.tfuxu.Halftone.json:47]:
```suggestion
        "config-opts": [
          "--with-heic=yes",
          "--with-rsvg=yes",
        ],
```

**Comment by tfuxu** [comment]
bot, build io.github.tfuxu.Halftone

**Pull request closed without merging**